web.test.tests/web.test.tests/Tests/DepositePageTests.cs

```diff
@@ -20,16 +19,20 @@ class DepositePageTests
         public void Setup()
         {
             driver = new ChromeDriver();
-
             driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(30);
             driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
-
             driver.Url = "http://localhost:64177/Login";
 
-            driver.FindElement(By.Id("login")).SendKeys("test");
-            driver.FindElement(By.Id("password")).SendKeys("newyork1");
-            driver.FindElement(By.Id("loginBtn")).Click();
+            new LoginPage(driver).Login("test", "newyork1");
+            new WebDriverWait(driver, TimeSpan.FromMilliseconds(10000)).Until(ExpectedConditions.VisibilityOfAllElementsLocatedBy(By.Id("amount")));
+
+            new DepositPage(driver).OpenSettingsPage();
 
+            SettingsPage settingsPage = new SettingsPage(driver);
+            settingsPage.SelectDateFormat.SelectByText("dd/MM/yyyy");
+            settingsPage.SelectNumberFormat.SelectByText("123,456,789.00");
+            settingsPage.SelectDefaultCurrency.SelectByText("$ - US dollar");
+            settingsPage.Save();
             new WebDriverWait(driver, TimeSpan.FromMilliseconds(10000)).Until(ExpectedConditions.VisibilityOfAllElementsLocatedBy(By.Id("amount")));
```

Comment:
These lines may be moved to 'SettingsPage' as 
public void ResetToDefault()
{
...
}

```suggestion
            new DepositPage(driver).OpenSettingsPage().ResetToDefault();
```

Reply:
Done